### PR TITLE
Added auto deployment for dashboards

### PR DESCRIPTION
**Pull request by ivo-ritense**

### CODE REVIEW

File: changelog/build.gradle [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    implementation project(":contract")
+
+    implementation "org.springframework.boot:spring-boot-starter-data-jpa"
+
+    implementation "io.github.microutils:kotlin-logging:${kotlinLoggingVersion}"
+
+    implementation "org.liquibase:liquibase-core:${liquibaseVersion}"
+
+    testImplementation (project(":core")) {
+        exclude(group: "com.ritense.valtimo", module: "authorization")
```
Comment (theo-ritense) [inline changelog/build.gradle:35]: This module does not exist here

File: app/gzac/src/main/resources/config/application.yml [hunk 2]
```diff
@@ -243,3 +243,5 @@ external:
             bezwaar:
                 caseKey: bezwaar
                 processDefinitionKey: bezwaar
+
+valtimo.pbac.clear-tables: true
```
Comment (theo-ritense) [inline app/gzac/src/main/resources/config/application.yml:247]: Not relevant in development

File: changelog/src/main/kotlin/com/ritense/valtimo/changelog/domain/Changeset.kt [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+    @Column(name = "id", nullable = false, updatable = false, unique = true)
+    val id: String,
+
+    @Column(name = "key")
+    val key: String?,
+
+    @Column(name = "filename", nullable = false, updatable = false)
+    val filename: String,
+
+    @Column(name = "dateexecuted", nullable = false, updatable = false)
```
Comment (theo-ritense) [inline changelog/src/main/kotlin/com/ritense/valtimo/changelog/domain/Changeset.kt:39]: It's not really a date though...:
`executed_on` ?

File: changelog/src/main/kotlin/com/ritense/valtimo/changelog/domain/Changeset.kt [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+    val dateExecuted: Instant,
+
+    @Column(name = "orderexecuted", nullable = false, updatable = false, unique = true)
```
Comment (theo-ritense) [inline changelog/src/main/kotlin/com/ritense/valtimo/changelog/domain/Changeset.kt:42]: I don't like the name:
`execution_order` ?

Comment (ivo-ritense) [reply changelog/src/main/kotlin/com/ritense/valtimo/changelog/domain/Changeset.kt:42]: I disagree. That to me would imply the order in which it should be executed. This variable shows the order in which all change sets were executed.

File: changelog/src/main/kotlin/com/ritense/valtimo/changelog/domain/Changeset.kt (hunk 3 above)
Comment (ivo-ritense) [reply changelog/src/main/kotlin/com/ritense/valtimo/changelog/domain/Changeset.kt:39]: It is in line with the way this kind of thing is named in Valtimo though.

File: changelog/build.gradle (hunk 1 above)
Comment (ivo-ritense) [reply changelog/build.gradle]: Removed

File: app/gzac/src/main/resources/config/application.yml (hunk 2 above)
Comment (ivo-ritense) [reply app/gzac/src/main/resources/config/application.yml]: Removed

**Review by theo-ritense — APPROVED** [review]

**Pull request merged**